provision-nginx.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+# shellcheck disable=1091
+. mail-toaster.sh || exit
+
+# shellcheck disable=2016
+export JAIL_CONF_EXTRA="
+		mount += \"$ZFS_DATA_MNT/webmail \$path/data nullfs rw 0 0\";"
+
+
+install_nginx()
+{
+	stage_pkg_install nginx dialog4ports || exit
+
+	local _nginx_conf="$STAGE_MNT/usr/local/etc/nginx/conf.d"
+	mkdir -p "$_nginx_conf" || exit
+
+	tee "$_nginx_conf/mail-toaster.conf" <<'EO_NGINX_MT6'
+set_real_ip_from 127.0.0.12;
+real_ip_header X-Forwarded-For;
+
+location / {
+   root   /usr/local/www/data;
+   index  index.html index.htm;
+}
```

Comment:
Since this jail will not possibly have a different httpd in it (the reason for switching to a generic dir), I'd suggest removing this block and leaving the default /usr/local/www/nginx directory.